docs/release-notes.md

```diff
@@ -10,6 +10,8 @@ nav_order: 9
 
 ### Features
 
+-- Support kubevirt cloud init no cloud
```

Comment:
This reads weird,

wdyt 

"- Add  support for`nocloud` config fetching in kubevirt cloud"

Reply:
Done